metadata.json

```diff
@@ -1,23 +1,79 @@
 {
-  "name": "mrpuppet-tagmail",
-  "version": "0.1.3",
-  "author": "mrpuppet",
-  "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags",
+  "name": "puppetlabs-tagmail",
+  "version": "1.0.0",
```

Comment:
Possibly. I forgot we had our own internal QA process for this stuff. A  < 1.0.0 release would be fine.  

--  
Tom Linkin
Professional Services Engineer
http://puppetlabs.com/
twitter: @trlinkin



On Monday, February 2, 2015 at 6:49 PM, Hunter Haugen wrote:

> In metadata.json (https://github.com/puppetlabs/puppetlabs-tagmail/pull/2#discussion_r23970618):
> > @@ -1,23 +1,79 @@ > { > - "name": "mrpuppet-tagmail", > - "version": "0.1.3", > - "author": "mrpuppet", > - "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags", > + "name": "puppetlabs-tagmail", > + "version": "1.0.0",  
> This hasn't gone through QA yet. Should we do something other than a 1.0.0 first until it gets reviewed?
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/puppetlabs/puppetlabs-tagmail/pull/2/files#r23970618).
>  
>  
>  
>  
>